src/ServiceControl.Audit.Persistence.RavenDB/IRavenPersistenceLifecycle.cs

```diff
@@ -6,5 +6,6 @@
     interface IRavenPersistenceLifecycle
     {
         Task Initialize(CancellationToken cancellationToken = default);
+        Task Stop(CancellationToken cancellationToken);
```

Comment:
```suggestion
        Task Stop(CancellationToken cancellationToken = default);
```

Reply:
I would like to understand this suggestion better, @danielmarbach.
I can understand suggesting such a change for a public nuget lib like our NServiceBus Nugets.
But for an app like ServiceControl, I can't see a good reason to suggest not passing the cancellation token, and that is what I have been doing in apps code.
Can you expand on why you think this is a good change for apps?